draft-ietf-quic-http.md

```diff
@@ -1839,18 +1839,22 @@ Specification:
 : An optional reference to a specification that describes the use of the
   setting.
 
+Default:
+: The value of the setting unless otherwise indicated. SHOULD be the most
+  restrictive possible value.
+
 The entries in the following table are registered by this document.
 
-| ---------------------------- | ------ | ------------------------- |
-| Setting Name                 |  Code  | Specification             |
-| ---------------------------- | :----: | ------------------------- |
-| Reserved                     |  0x2   | N/A                       |
-| Reserved                     |  0x3   | N/A                       |
-| Reserved                     |  0x4   | N/A                       |
-| Reserved                     |  0x5   | N/A                       |
-| MAX_HEADER_LIST_SIZE         |  0x6   | {{settings-parameters}}   |
-| NUM_PLACEHOLDERS             |  0x9   | {{settings-parameters}}   |
-| ---------------------------- | ------ | ------------------------- |
+| ---------------------------- | ------ | ------------------------- | ------- |
+| Setting Name                 |  Code  | Specification             | Default |
+| ---------------------------- | :----: | ------------------------- | ------- |
+| Reserved                     |  0x2   | N/A                       | N/A     |
+| Reserved                     |  0x3   | N/A                       | N/A     |
+| Reserved                     |  0x4   | N/A                       | N/A     |
+| Reserved                     |  0x5   | N/A                       | N/A     |
+| MAX_HEADER_LIST_SIZE         |  0x6   | {{settings-parameters}}   | 0       |
```

Comment:
This is value is defined elsewhere to be unlimited, RFC 7540 used (infinite) but I'm not fussy.

Reply:
If you want to make SETTINGS REALLY required, making this default to 0 would certainly do it :)

Reply:
Well, not really as endpoints are allowed to ignore this value :)

But yeah https://quicwg.org/base-drafts/draft-ietf-quic-http.html#rfc.section.7.2.5.1 says the default value is unlimited, so this should be aligned to that (though I'd much rather remove this setting completely as it serves no purpose...).

Reply:
Bah, yes.  I was just following "SHOULD be the most restrictive possible value" on autopilot, but this is an exception (because it's advisory).

Reply:
Would 2^62-1 be less offensive?